<launch>

  <node pkg="rover23_localization" type="fake_odom_tf.py" name="fake_odom_tf_publisher" output="screen">
  </node>

  <!-- <node pkg="rover23_localization" type="fake_map_tf.py" name="fake_odom_publisher" output="screen">
  </node> 
   -->
</launch>